packages/office-ui-fabric-react/src/components/Persona/PersonaPresence/PersonaPresence.base.tsx

```diff
@@ -67,7 +68,7 @@ export class PersonaPresenceBase extends BaseComponent<IPersonaPresenceProps, {}
     }
 
     return (
-      <div className={classNames.presence} style={coinSizeWithPresenceStyle}>
+      <div className={classNames.presence} style={coinSizeWithPresenceStyle} {...presenceTitle && { title: presenceTitle }}>
```

Comment:
>{...presenceTitle && { title: presenceTitle }} [](start = 77, length = 46)

Let's assign this to a const (that defaults to undefined) first and then set it here to avoid using spread for a single attribute